app/build.gradle.kts

```diff
@@ -58,27 +58,27 @@ apollo {
 }
 
 dependencies {
-    implementation("androidx.core:core-ktx:1.7.0")
-    implementation("androidx.compose.ui:ui:1.1.1")
-    implementation("androidx.compose.material:material:1.1.1")
-    implementation("androidx.compose.ui:ui-tooling-preview:1.1.1")
-    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.4.1")
-    implementation("androidx.activity:activity-compose:1.4.0")
+    implementation(AndroidX.core.ktx)
+    implementation(AndroidX.compose.ui)
+    implementation(AndroidX.compose.material)
+    implementation(AndroidX.compose.ui.toolingPreview)
+    implementation(AndroidX.lifecycle.runtimeKtx)
+    implementation(AndroidX.activity.compose)
 
     // Apollo Kotlin
-    implementation("com.apollographql.apollo3:apollo-runtime:3.1.0")
+    implementation("com.apollographql.apollo3:apollo-runtime:_")
```

Comment:
Already created an issue for this! Remember to work on the assigned [plugin issue](https://github.com/jmfayard/refreshVersions/issues/476) and see if we can group plugin and runtime versions (as it was before, with `$apollo_version` in the `ext` block).